runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/Actions/general.actions.ts

```diff
@@ -13,13 +13,14 @@ const getGeneralActions : () => Record<string, IBpmnAction> = () => ({
                 type: 'object',
                 properties: {
                     input: {
-                        title: translate('Process.Details.Modeler.Actions.General.ConsoleLog.Input'),
-                        type: 'object',
+                        title: '',
```

Comment:
if empty so maybe this field is redundant?

Reply:
Unfortunately, I haven't found the way to remove the title and not get title doubled or error -> if title removed than default title is displayed:

I set title to "null" instead of ' '. Is it better?